docs/en/operations/external-authenticators/ssl-x509.md

```diff
@@ -0,0 +1,24 @@
+# SSL X.509 certificate authentication {#ssl-external-authentication}
+
+[SSL 'strict' option](../server-configuration-parameters/settings.md#server_configuration_parameters-openssl) enables mandatory certificate validation for the incoming connections. In this case, only connections with trusted certificates can be established. Connections with untrusted certificates will be rejected. Thus, certificate validation allows to uniquely authenticate an incoming connection. `Common Name` field of the certificate is used to identify connected user. This allows to associate multiple certificates with the same user. Additionally, reissuing and revoking of the certificates does not affect the ClickHouse configuration.
```

Comment:
It's not very obvious that you're talking about `verificationMode`, let's write about that. Also I think before talking about configuration it's worth telling what this authentication is and that it's used when connecting to Clickhouse server only via https protocol.